pkg/config/pulse_server.go

```diff
@@ -6,15 +6,17 @@ import (
 
 // PulseServerConfig contains the configuration for the maestro pulse server.
 type PulseServerConfig struct {
-	PulseInterval int64 `json:"pulse_interval"`
-	CheckInterval int64 `json:"check_interval"`
+	PulseInterval           int64 `json:"pulse_interval"`
+	CheckInterval           int64 `json:"check_interval"`
+	EnableConsistentHashing bool  `json:"enable_consistent_hashing"`
 }
 
 // NewPulseServerConfig creates a new PulseServerConfig with default 10 second pulse and check intervals.
 func MewPulseServerConfig() *PulseServerConfig {
 	return &PulseServerConfig{
-		PulseInterval: 10,
-		CheckInterval: 10,
+		PulseInterval:           10,
```

Comment:
setting check and pulse to the same value is not quite safe. You might consider a the pulse to be 3~5 times of the check.

Reply:
how about we only expose check interval, and make pulse interval 3~5 times of the check interval?

Reply:
Updated this, now we only expose pulse interval. maestro check the heartbeat each pulseInterval/3 seconds.